backend/src/main/scala/cromwell/backend/BackendLifecycleActorFactory.scala

```diff
@@ -24,6 +30,19 @@ trait BackendLifecycleActorFactory {
                              initializationData: Option[BackendInitializationData],
                              serviceRegistryActor: ActorRef): Props
 
+  /**
+    * The default implementation of this method ignores the fact that there has been a cache hit and creates props for
+    * a job execution actor.  If a backend has implemented call caching this will not be the desired behavior and this
+    * method should be overridden.
```

Comment:
i.e. will it try to run once and fail, then fall back to running it again, and fail again?